sites/platform/src/create-apps/app-reference.md

```diff
@@ -311,21 +311,21 @@ See how to [troubleshoot the warning](./troubleshoot-mounts.md#overlapping-folde
 | Name          | Type                 | Required | Description |
 | ------------- | -------------------- | -------- | ----------- |
 | `source`      | `local`, `service`, or `tmp` | Yes      | Specifies the mount type. Set to: </br> - `local` so your mount is unique to the app (requires `disk` to be set for the app).</br> - `service` so your [Network Storage](/add-services/network-storage.md) mount can be shared between several apps.</br> - `tmp` to mount a directory within the `/tmp` directory of your app. `tmp` mounts are **local ephemeral mounts**; their content may be removed during infrastructure maintenance operations. They allow you to **store files that you're not afraid to lose**, such as your application cache that can be seamlessly rebuilt.</br>Note that the `/tmp` directory has a [maximum allocation of 8 GB](/create-apps/troubleshoot-disks.md#no-space-left-on-device). |
-| `source_path` | `string`             | Yes      | The subdirectory within the mounted disk (the source) where the mount should point. If an empty string is passed, points to the entire directory. |
-| `service`     | `string`             |          | The name of the [network storage service](../add-services/network-storage.md). |
+| `source_path` | `string`             | No      | The subdirectory within the mounted disk (the source) where the mount should point.</br> If you don't define a `source_path`, or if an empty string is passed, the mount will point to [the entire {{< variable "DIRECTORY" >}}](#mounts). |
```

Comment:
We need to also add an important warning because if a user changes the mount name while `source_path` was ignored, then the mount will be empty since the `source_path` will be changed accordingly with the new mount name, thus pointing a new empty shared directory.
The data won't be lost though. To ensure continuity, the user must specify the `source_path` in this case.

Example: I define a mount on `/var/storage`, then rename it to `/foo`. In order to still get access to my files, I need to define the `source_path` to `var/storage`.

Before:
```yaml
    mounts:
      "/var/storage":
        source: storage
```

After:
```yaml
    mounts:
      "/foo":
        source: storage
        source_path: var/storage
```

Note that the `source_path` is identical to the previous mount name, without the leading `/` (and potentially the trailing one if it's present)